index.html

```diff
@@ -48,5 +48,165 @@ <h1>Hey there. I’m Imran Hossain <br> <label>I’m a software developer</label
+                    <ul class="laguages">
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>JavaScript</li>
+                        <li>html</li>
+                    </ul>
+                    <button class="btm" href="#">See Project</button>
+                </div>
+            </li>
+            <!-- card 3 -->
+            <li class="card">
+                <div class="preview">
+
+                </div>
+                <div class="details">
+                    <a class="project_tle" href="#" >Multi-Post Stories <br> Gain+Glory</a>
+                    <ul class="laguages">
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>JavaScript</li>
+                        <li>html</li>
+                    </ul>
+                    <button class="btm" >See Project</button>
+                </div>
+            </li>
+            <!-- card 4 -->
+            <li class="card">
+                <div class="preview">
+
+                </div>
+                <div class="details">
+                    <a class="project_tle" href="#" >Multi-Post Stories <br> Gain+Glory</a>
+                    <ul class="laguages">
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>JavaScript</li>
+                        <li>html</li>
+                    </ul>
+                    <button class="btm" >See Project</button>
+                </div>
+            </li>
+            <!-- card 5 -->
+            <li class="card">
+                <div class="preview">
+
+                </div>
+                <div class="details">
+                    <a class="project_tle" href="#" >Multi-Post Stories <br> Gain+Glory</a>
+                    <ul class="laguages">
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>JavaScript</li>
+                        <li>html</li>
+                    </ul>
+                    <button class="btm" >See Project</button>
+                </div>
+            </li>
+            <!-- card 6 -->
+            <li class="card">
+                <div class="preview">
+
+                </div>
+                <div class="details">
+                    <a class="project_tle" href="#" >Multi-Post Stories <br> Gain+Glory</a>
+                    <ul class="laguages">
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>JavaScript</li>
+                        <li>html</li>
+                    </ul>
+                    <button class="btm" >See Project</button>
+                </div>
+            </li>
+        </ul>
+    </section>
+    <!-- About me Section -->
+    <section class="about">
+        <div class="aboutArticle">
+            <h1>About me</h1>
+            <p>
```

Comment:
- [ ] Please kindly change the `h1` tag to any other header tags (`h2, h3....`). It's part of best practises to only have one `h1` tag in a HTML document possibly because of SEO optimization. 👍

Reply:
@OyePriscilla I have change h1 to h2 , Can you kind explain more about "responsive until the desktop screen size" part?

Reply:
@imranhossainemi Responsive websites are websites that adapt to all screen sizes.